src/modules/better_viewer_list/index.js

```diff
@@ -255,6 +254,10 @@ class BetterViewerListModule {
         // Load viewers
         this.loadViewerList();
     }
+
+    capitalize(str) {
```

Comment:
this doesn't really need to be on the class.. i'd drop it as a named function at the top